main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"challenge2/ipScanner"
+	"log"
+	"net/http"
+)
+
+func main() {
+	http.HandleFunc("/ip/details", ipScanner.StartIPScan)
```

Comment:
@rdiazrovira well done, i think you missed the part were you should handle non-existing routes with a json 404 response, could you implement it please ?